src/Sylius/Bundle/ShippingBundle/Resources/translations/messages.en.yml

```diff
@@ -50,6 +50,7 @@ sylius:
         shipping_category:
             description: Description
             name: Name
+            code: Code
```

Comment:
Let's move the translation under `sylius.ui.code` in the WebBundle and don't translate it several times as it is done now for most of translations :)